### DISCUSSION TITLE
Problems after update to 20220610

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#940] [Fixed Issues]
I updated to 20220610 last night on my test RPi3A and all seemed OK. I then tried to configure CamillaDSP to see what this was all about which again seemed to go OK.  I then tried to configure CamillaDSP from the Main menu option  at which point I got a very small config screen which didn't mean a lot to me. I exited out  and then the PI just went to a black screen.  Checking this morning it seems like (at least) all the .sh files in /srv/http/bash have been saved as binary (?) files or worse still encrypted. What has happened here ?  The web app is completely broken and still only gives a black screen.  I guess the only way out of this mess is to reinstall ?  I will save the original SD card in case you need any information from it.

BTW - what is Camilla DSP and how / when should it be used ?  Does it require any specific hardware ? Is this the cause of the failure above because I have a bog standard RPi 3A running the 64 bit version of rAudio-1 via Soundcore2 BT speakers.

Chris

**Comment by R101G** [comment]
I am very pleased to say that I don't see this feature anywhere. Presumably because I have selected bit-perfect output?

**Reply by ceejayemm** [reply]
Bit Perfect doesn't seem to be available when using BT output.  I am glad to report that a reinstallation and update to 20220610 WITHOUT enabling CamillaDSP seems to be working perfectly.

Chris

**Comment by AndyFx12** [comment]
Small CamillaDSP-config screen could be a colors-CSS-thing. Try:
 ln -sf /srv/http/assets/css/colors.css /srv/http/settings/camillagui/build
 ln -sf /srv/http/assets/img/icon.png /srv/http/settings/camillagui/build
(install.sh should do this - but didn´t in my case)

With CamillaDSP you can add audio-filters like 
![grafik](https://user-images.githubusercontent.com/35779271/173189595-c5d822b8-15fd-45cb-90d3-ee125b2bc4ba.png)
Many options will be available.

IMHO no special hardware is required, because of using alsa-loopback. I don´t know, if it works through bluealsa, though.

Andy

**Reply by ceejayemm** [reply]
Thanks Andy but I may have misled you when I said the config screen was small. What I should have said was it contained only a few items. I can't remember them all now but it was only 2, or may be 3, items - one of which was a time in 'ms' which had the word 'default' in the box but I can't remember the rest. It wasn't anything like your snippet above.

Chris

**Reply by AndyFx12** [reply]
Hi Chris,
screenshot show´s working CamillaDSP (to show, what Camilla can do) - the "color-css-issue" produce some invisible buttons. Only some words are visible. So I think your config screen looked like this.
Andy
